pkg/controller/certificates/sync.go

```diff
@@ -57,8 +58,14 @@ const (
 	messageIssuingCertificate  = "Issuing certificate..."
 	messageRenewingCertificate = "Renewing certificate..."
 
+	infoCertificateDuration = "WarnCertificateDuration"
+	infoScheduleModified    = "WarnScheduleModified"
+
```

Comment:
These two vars are unused

Reply:
Another holdover from the event triggers, removed